.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -41,7 +41,7 @@ By following these guidelines, your PRs are more likely to be merged quickly aft
     black .
     isort .
     mypy
-    autoflake --remove-all-unused-imports --recursive --ignore-init-module-imports autogpt tests --in-place
+    autoflake --remove-all-unused-imports --recursive --ignore-init-module-imports --ignore-pass-after-docstring autogpt tests --in-place
```

Comment:
Adding this flag was necessary because, it turns out, every single plugin will not pass `autoflake`. See https://github.com/PyCQA/autoflake/pull/143 for details.